hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleSchemaRegistry.java

```diff
@@ -248,8 +251,8 @@ public void migrate(
                                 && previousVersion != null
                                 && alreadyIncludesStateDefs(previousVersion, s.getVersion()))
                         .toList();
-                final var redefinedWritableStates =
-                        applyStateDefinitions(schema, schemasAlreadyInState, config, metrics, stateRoot);
+                final var redefinedWritableStates = applyStateDefinitions(
+                        schema, schemasAlreadyInState, nodeConfiguration, platformConfiguration, metrics, stateRoot);
```

Comment:
`platformConfiguration` is nullable here but in `applyStateDefinitions` method it's annotated as `@NonNull`. Please, pick one.

Reply:
Good catch: https://github.com/hashgraph/hedera-services/pull/15562/commits/c67d4f2b6e9de8f4342fb1896596b61d7a1b9ddc